pkg/handler/processor/process/process.go

```diff
@@ -228,14 +237,14 @@ func decodeDocument(ctx context.Context, i *processor.Document) error {
 	case processor.EncodingUnknown:
```

Comment:
should this case processor.EncodingUnknown:` as the document is not processable by GUAC (or log that it is being skipped because it does not understand the encoding?)

Reply:
@pxp928  Before this pr one of the error messages (more than one in the stacktrace) with `guacone collect files yourfile.json.bz2` file was` "msg":"emit error: unable to ingest document: unable to process doc: no document processor registered for type: UNKNOWN, format: UNKNOWN, document: UNKNOWN"`

Reply:
so in that case, if the case is not being used, we can just remove it.

Reply:
> so in that case, if the case is not being used, we can just remove it.

Ok